### ISSUE TITLE
Allow marking modpack as server pack

### ISSUE THREAD

**Issue by natanfudge** [itsmeow/curseforge-upload#22]
See https://github.com/Krutoy242/Enigmatica2Expert-Extended/issues/227

**Comment by Krutoy242** [comment]
As far as I know, CF API doesn't have this possibility at all and can be done only manually.

**Comment by itsmeow** [comment]
I'll look into getting Overwolf to add support for this. They are fairly good about this type of stuff now.

**Comment by itsmeow** [comment]
Upvote this idea for traction https://curseforge-ideas.overwolf.com/ideas/CF-I-5068

**Comment by Krutoy242** [comment]
> Upvote this idea for traction [curseforge-ideas.overwolf.com/ideas/CF-I-5068](https://curseforge-ideas.overwolf.com/ideas/CF-I-5068)

Sorry, I can't

![image](https://github.com/itsmeow/curseforge-upload/assets/936618/c9dc1592-5273-452f-851f-a649d58a72ae)

**Comment by itsmeow** [comment]
Yeah moderators have to approve it first or something. Dumb CF stuff

**Comment by itsmeow** [comment]
It is approved now so it can be upvoted

**Comment by hk21702** [comment]
This might already be supported? I'm looking at other actions and some have the field isServerPack as part of the json object being sent as part of the HTTP post request. I didn't test it myself though. https://github.com/Xikaro/upload-curseforge-modpack-action/tree/master

![image](https://github.com/user-attachments/assets/0cc277ef-d279-424f-a6de-8543fcc67af5)

**Comment by hk21702** [comment]
> This might already be supported? I'm looking at other actions and some have the field isServerPack as part of the json object being sent as part of the HTTP post request. I didn't test it myself though. https://github.com/Xikaro/upload-curseforge-modpack-action/tree/master
> 
> ![image](https://private-user-images.githubusercontent.com/28567881/353953670-0cc277ef-d279-424f-a6de-8543fcc67af5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI0NTMyODQsIm5iZiI6MTcyMjQ1Mjk4NCwicGF0aCI6Ii8yODU2Nzg4MS8zNTM5NTM2NzAtMGNjMjc3ZWYtZDI3OS00MjRmLWE2ZGUtODU0M2ZjYzY3YWY1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzMxVDE5MDk0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTMzMGYxZWY1ZGVjNzMxY2I5MWI5ZTZmZDg3MDM3YTU1YmU2ZTQ2YjM1NDY3N2JmMjU0YjUxZWM5OTQzZTliMmMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.W2Im1mwFpsoLxliW7yn6FFdiOaqomfBmlmURpf2XLlc)

Just tested that action. It seems to work? It sets the server file as a child of the main file being uploaded.

**Comment by Krutoy242** [comment]
> Just tested that action. It seems to work? It sets the server file as a child of the main file being uploaded.

It should not just sets as a child but mark it as "server pack"

![image](https://github.com/user-attachments/assets/321dc7f9-a64c-4224-aac4-3ce9d833939e)
